pkgs/games/abuse/abuse.sh

```diff
@@ -0,0 +1,18 @@
+#! @shell@
+
+if grep datadir ~/.abuse/abuserc &>/dev/null; then
```

Comment:
Is grep part of the standard env? I am not sure.

Reply:
Good catch, I don't know. How could I find out? I'm having trouble finding the default set of packages.

Reply:
I now checked. grep is in the set of required packages ([nixos/modules/config/system-path.nix]). So I consider this issue to be resolved. :-)